marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-common-writer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "role-name": "data-hub-common-writer",
+  "description": "Contains a common set of privileges/roles needed for Data Hub 5+ write operations",
+  "role": [
+    "data-hub-common",
+    "rest-writer"
```

Comment:
do we need to inherit data-hub-module-writer here?

Reply:
no. that isn't a common write permission that we want to grant to any user what has write privileges